lib/hatchet/app.rb

```diff
@@ -108,8 +108,8 @@ def add_database(plan_name = 'heroku-postgresql:dev', match_val = "HEROKU_POSTGR
     # but programatically and with more control
     def run(cmd_type, command = nil, options = {}, &block)
       command        = cmd_type.to_s if command.nil?
-      heroku_options = (options.delete(:heroku) || {}).map {|k,v| "--#{k.to_s.shellescape}=#{v.to_s.shellescape}"}.join(" ")
-      heroku_command = "heroku run #{command.to_s.shellescape} -a #{name} #{ heroku_options }"
+      heroku_options = (options.delete(:heroku) || {}).map {|k,v| "--#{k.to_s.shellescape}".concat(v ? "=#{v.to_s.shellescape}" : "")}.join(" ")
```

Comment:
I would prefer if we turned this into multi line logic over the ternary operator

```ruby
heroku_options = (options.delete(:heroku) || {}).map do |k,v|
  arg = "--#{k.to_s.shellescape}"
  arg << "=#{v.to_s.shellescape}" unless v.nil?
  arg
end.join(" ")